tests/src/Kernel/ListsQueryTest.php

```diff
@@ -158,6 +284,47 @@ public function testQuerySorting(): void {
     ], $titles);
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  protected function createTranslatedTestContent(string $bundle, array $values = []): void {
```

Comment:
We can probably just refactor the createTestContent method to accept a list of language for which to create translations.